### ISSUE TITLE
Difference between eap-generic and eap-config-full is still confusing for admins

### ISSUE THREAD

**Issue by jornane** [GEANT/CAT#279]
# Issue type
- Questions about the eduroamCAT or its usage should be sent to the [users mailing list](https://lists.geant.org/sympa/info/cat-users).

- [ ] Defect - Crash/memory corruption.
- [ ] Defect - Non-compliance with a standards document or incorrect OS API usage.
- [X] Defect - Unexpected behaviour (obvious or has been verified by a project member).
- [ ] New feature request.

# Defect/Feature description

## How to reproduce issue

1. Go to the admin portal, go to the "Installer Fine-Tuning and Download" page for a PEAP or TTLS profile, that has an RCOI set.
2. Observe there is an EAP config device there.
3. Click the `Download eduroam® Installer` button.
4. Try to load the resulting .eap-config file in geteduroam.
5. Observe that Passpoint is not configured.
 
## Detail of issue

This is the same issue as #271, which was closed without fully resolving it, and which is not replied to anymore. In #271, the "EAP config" profile was removed from the user view, but it is still in the admin view.

Administrators know that geteduroam uses eap-config profiles under the hood, so in order to test their CAT profiles, they download the .eap-config file from CAT and try to load it in the app, which is a supported feature of the geteduroam app on all platforms.

## Recommended solution

* Add a device to the "Installer Fine-Tuning and Download" list, geteduroam, which is eap-generic.
* Possibly remove the "EAP config" (eap-config-full) device type from the admin interface as well; the format is not supported by any current application, and as such is of little use to admins.

Possibly add a device type which is specificly for geteduroam, in the same fashion as the device types `android_4_7` and `android_recent`. In fact, `android_recent` refers to Android 8 and up, which is also supported by geteduroam.

**Comment by twoln** [comment]
Absolutely, and actually this probably comment does not belong to GitHub at all. This is not a coding but purely configuration issue. When preparing the new release for production I will ask Jorn to check that things look and work as he would expect.

**Comment by twoln** [comment]
Currently eep-generic has the following options set:

- compatibility for eduroamCAT - probably should be removed
- single language - probably correct
- OpenRoaming support - probably to be removed
The title for this button should then be either "geteduroam" or "geteduroam profile". The second one would require a translation, but since this is admin area we can just let it go. I would be against just "geteduroam" as this could be misleading as pointing to the app itself. 
What about the icon for this - I guess the geteduroam one would be just fine?
Finally - should we wait for the new release - I would thing that this is reasonable - people expect changes in such moments.

**Comment by twoln** [comment]
I have removed eep-config-full from the admin download as well. But I will have more requests for clarification later.
